account_asset_management/models/account_asset.py

```diff
@@ -259,6 +259,14 @@ class AccountAsset(models.Model):
         states=READONLY_STATES,
         check_company=True,
     )
+    depreciation_line_manual_ids = fields.One2many(
```

Comment:
Why do you need this field if you already have depreciation_line_ids?

Reply:
Great spot, maybe that one can be removed. I'll check

Reply:
Removed the field :+1: